Lint-Whoppers/nutshell
# Sp EDIT and DELETE for Messages Component

# Description

Implemented full CRUD for Messages Component.

Fixes https://github.com/Lint-Whoppers/nutshell/issues/21 https://github.com/Lint-Whoppers/nutshell/issues/22

## Type of change

Please delete options that are not relevant.


# Testing Instructions

1) `git fetch --all`
2) `git checkout sp-editmessage`
3)  Check to see if you can Create, Edit, and Delete messages. **(Edit and Delete buttons will ONLY show up if you are the author of the message)**



# Checklist:
